openhtf/__init__.py

```diff
@@ -164,21 +225,14 @@ def plug_type_map(self):
     return plug_type_map
 
 
-def CreateArgParser():
+def CreateArgParser(add_help=True):
```

Comment:
Actually I meant to reverse this and make `add_help` default to False and override it to True in Execute, this is intended to provide the user with a means to include OpenHTF flags in their own flags' help docs.  So they could do:
```
my_parser = argparse.ArgumentParser('My Test arg parser', parents=[openhtf.CreateArgParser()])
```